js/src/features/human-resouce/components/HumanResourceCalendar/helpers.ts

```diff
@@ -67,3 +67,16 @@ export function getAbsencesDays(absences: Absence[], reason: AbsenceReason) {
 export function getCapitalLetter(value: string | null) {
```

Comment:
Strange function name. How it will be work if my name `dmitry yakubousky`? The sense of this function is just to take first letter, but not capitalize it.

Reply:
I think you should create function `getEmplyeeInitials` which will be more usefull.

Reply:
Done.

Reply:
Nop

Reply:
Done